### PR TITLE
Type hints for scripts

### PR DESCRIPTION
**Pull request by rchan26**
Partially addressing #526 - added type hints for `fill_*` and `make_*` scripts in `airsenal/scripts`.

### CODE REVIEW

File: airsenal/scripts/fill_fixture_table.py [hunk 1]
```diff
@@ -87,12 +92,12 @@ def fill_fixtures_from_api(season, dbsession=session):
             raise ValueError(f"Can't find team(s) with id(s): {away_id}")
         if not update:
             dbsession.add(f)
-
     dbsession.commit()
-    return True
```
Comment (rchan26) [inline airsenal/scripts/fill_fixture_table.py:92]: Seems like this bool is never used anywhere

**Comment by rchan26** [comment]
Had a go at doing type hints for each function in `scripts`. Hopefully got all the types correct...

File: airsenal/scripts/fill_fixture_table.py (hunk 1 above)
Comment (jack89roberts) [reply airsenal/scripts/fill_fixture_table.py:92]: It looks like there are a few places we `return True` (or False) without doing anything with them. They could all be removed as far as I'm concerned.

**Comment by jack89roberts** [comment]
Thanks @rchan26 - looks good to me from the bits I've looked through! See my comment above about `return True` but happy for this to be merged to develop without that (could be a separate issue) - up to you.

**Comment by rchan26** [comment]
Going to just merge this into develop - I only deleted one of the cases where the `return True` is not used after. I'll write an issue for this for us to clean them up at some other point 🙂

**Pull request merged**